esp32-led-blink-sdk/README.md

```diff
@@ -0,0 +1,39 @@
+# esp32-led-strip-sdk
```

Comment:
I think this name is copy-pasted, could you update it?

Reply:
sure, done